include/sqlpp11/update_list.h

```diff
@@ -279,7 +287,15 @@ namespace sqlpp
           -> _new_statement_t<consistent_t, update_list_t<Database, Assignments...>>
       {
         return {static_cast<const derived_statement_t<Policies>&>(*this),
-                update_list_data_t<Database, Assignments...>{assignments...}};
+                update_list_data_t<Database, Assignments...>{std::forward_as_tuple(assignments...)}};
+      }
+
+      template <typename Database, typename... Assignments>
+      auto _set_impl(consistent_t /*unused*/, std::tuple<Assignments...> assignments) const
```

Comment:
gcc on Travis seems to be confused by this version of `_set_impl` and the "old" one when facing a single assignment.

Smells like a compiler bug to me.

I believe that we could work around this by keeping only this version (i.e. call `make_tuple` in `set` in case of individual assignments.